ush/global_nest_patch.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+# shellcheck disable=SC2155,SC2312
+HOMEgfs=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/.." && pwd -P)
+declare -rx HOMEgfs
+
+## patch config.base
+sed -i -e 's#export BASE_CPLIC=.*#export BASE_CPLIC="/scratch2/BMC/wrfruc/Guoqing.Ge/ufs-ar/ICS/global-nest"#' -e 's/export DO_NEST="NO"/export DO_NEST="YES"/' $HOMEgfs/parm/config/gfs/config.base
```

Comment:
This should be set by the setup_workflow script via a --nest option. Including a script to patch the scripts makes code management much harder. Relying on commented-out code is a bad practice.

Get rid of it and put in my original changes to setup_workflow.

Reply:
Do you mean we automatically set DO_NEST=YES and a new BASE_CPLIC location once we specify the "--nest" option?

I think whether to do nesting can be determined by a user through manually setting DO_NEST=YES in config.base under $EXPDIR

However, yes, we can make it happen automatically by the --nest option if needed.

Reply:
I'd prefer to reduce the manual edits of config.* files as much as possible.